docs/eventing/broker-trigger.md

```diff
@@ -13,29 +13,63 @@ A Broker represents an 'event mesh'. Events are sent to the Broker's ingress and
 are then sent to any subscribers that are interested in that event. Once inside
 a Broker, all metadata other than the CloudEvent is stripped away (e.g. unless
 set as a CloudEvent attribute, there is no concept of how this event entered the
-Broker).
+Broker). There can be different classes of `Broker`s providing different kinds
```

Comment:
This should probably be a new paragraph. I'd drop the code tags around `Broker` since the previous uses of Broker don't have them.